.gitignore

```
@@ -42,8 +42,8 @@ coverage/
 /out
 
 # various stuff that VC++ produces/uses and is not in /out
-/Debug/
-/Release/
+**/*/Debug/
+**/*/Release/
```

1. the simpler sulution is
```suggestion
Release/
Debug/
```
2. If we do want to revert is back it probably should be:
https://github.com/nodejs/node/blob/0a25ace9c35b62ece4d32fd90b326d8063265109/.gitignore#L45-L48

---

Thanks @refack, and mine can also has the same function as yours, changed and I'll submit it because I want to make it more strict in rules.